src/Areas/AiFoundry/AiFoundrySetup.cs

```diff
@@ -0,0 +1,80 @@
+        model.AddCommand("describe", new Commands.ModelDescribeCommand(loggerFactory.CreateLogger<Commands.ModelDescribeCommand>()));
+
+        // Create deployment subgroup
+        var deployment = new CommandGroup("deployment", "Model deployment operations - Commands for managing and monitoring model deployments in AI Foundry projects.");
+        aiFoundry.AddSubGroup(deployment);
+
+        // Register deployment commands
+        deployment.AddCommand("list", new Commands.DeploymentListCommand(loggerFactory.CreateLogger<Commands.DeploymentListCommand>()));
+        deployment.AddCommand("describe", new Commands.DeploymentDescribeCommand(loggerFactory.CreateLogger<Commands.DeploymentDescribeCommand>()));
+
+        // Create connection subgroup
+        var connection = new CommandGroup("connection", "Project connection operations - Commands for managing external service connections and integrations in AI Foundry projects.");
+        aiFoundry.AddSubGroup(connection);
+
+        // Register connection commands
+        connection.AddCommand("list", new Commands.ConnectionListCommand(loggerFactory.CreateLogger<Commands.ConnectionListCommand>()));
+        connection.AddCommand("describe", new Commands.ConnectionDescribeCommand(loggerFactory.CreateLogger<Commands.ConnectionDescribeCommand>()));
+
+        // Create agent subgroup
+        var agent = new CommandGroup("agent", "AI agent operations - Commands for managing and configuring AI agents and assistants in AI Foundry projects.");
+        aiFoundry.AddSubGroup(agent);
+
+        // Register agent commands
+        agent.AddCommand("list", new Commands.AgentListCommand(loggerFactory.CreateLogger<Commands.AgentListCommand>()));
+        agent.AddCommand("describe", new Commands.AgentDescribeCommand(loggerFactory.CreateLogger<Commands.AgentDescribeCommand>()));
+
+        // Create dataset subgroup
+        var dataset = new CommandGroup("dataset", "Dataset operations - Commands for managing and working with datasets in AI Foundry projects.");
+        aiFoundry.AddSubGroup(dataset);
+
+        // Register dataset commands
+        dataset.AddCommand("list", new Commands.DatasetListCommand(loggerFactory.CreateLogger<Commands.DatasetListCommand>()));
+        dataset.AddCommand("describe", new Commands.DatasetDescribeCommand(loggerFactory.CreateLogger<Commands.DatasetDescribeCommand>()));
+
+        // Create vector store subgroup
+        var vectorStore = new CommandGroup("vectorstore", "Vector store operations - Commands for managing and working with vector stores in AI Foundry projects.");
+        aiFoundry.AddSubGroup(vectorStore);
+
+        // Register vector store commands
+        vectorStore.AddCommand("list", new Commands.VectorStoreListCommand(loggerFactory.CreateLogger<Commands.VectorStoreListCommand>()));
+        vectorStore.AddCommand("describe", new Commands.VectorStoreDescribeCommand(loggerFactory.CreateLogger<Commands.VectorStoreDescribeCommand>()));
+    }
+} 
```

Comment:
@hallipr I thought dotnet format would catch this but we have it defined to include a new line end of file in our config. 

https://github.com/Azure/azure-mcp/blob/main/.editorconfig#L12